monitor/getBalances.js

```diff
@@ -83,6 +83,28 @@ async function main(bridgeMode) {
     const foreignBridge = new web3Foreign.eth.Contract(FOREIGN_ERC_TO_NATIVE_ABI, COMMON_FOREIGN_BRIDGE_ADDRESS)
     const erc20Address = await foreignBridge.methods.erc20token().call()
     const erc20Contract = new web3Foreign.eth.Contract(ERC20_ABI, erc20Address)
+    let foreignHalfDuplexErc20Balance = 0
+    let displayHalfDuplexToken = false
```

Comment:
Do you think it makes sense to implement an opposite logic: first check if the bridge supports half duplex token and only after that check if the token swap allowed?

Reply:
Yes, I think it's better in the way you described. Updated the logic https://github.com/poanetwork/tokenbridge/pull/245/commits/22b1bf162377f7033d40ada2ba4a7f2866a583a8